src/components/ContentSidebar/SidebarNotices.js

```diff
@@ -7,46 +7,70 @@
 import React from 'react';
 import getProp from 'lodash/get';
 import SharedLinkExpirationNotice from 'box-react-ui/lib/features/item-details/SharedLinkExpirationNotice';
-import SidebarSection from './SidebarSection';
+import ItemExpirationNotice from 'box-react-ui/lib/features/item-details/ItemExpirationNotice';
 import { addTime } from '../../util/datetime';
 import type { BoxItem } from '../../flowTypes';
 import DateField from '../Date';
 
 const ONE_MINUTE_IN_MS = 60000;
 
+const NOTICE_DATE_FORMAT = {
+    month: 'short',
+    day: 'numeric',
+    year: 'numeric'
+};
+
 type Props = {
     file: BoxItem
 };
 
+/**
+ * Adds one minute to a ISO formatted date string
+ *
+ * @private
+ * @param {string} expiration - an item or shared link expiration
+ * @return {string} an ISO formatted date string
+ */
+function addMinuteToExpiration(expiration: string) {
```

Comment:
Probably move this and https://github.com/box/box-ui-elements/blob/master/src/util/datetime.js

into

box-react-ui/blob/master/src/utils/datetime.js

And reference that

Reply:
I'm up for moving the datetime.js stuff. But this method is specific to adding 1 minute for item expiration. The DateTime utils will already have generic time adding.

Reply:
Ported them over, will remove from here once that's merged